client/webserver/site/src/js/markets.js

```diff
@@ -1313,14 +1312,10 @@ export default class MarketsPage extends BasePage {
     }
     if (!this.validateOrder(order)) return
     const res = await postJSON('/api/trade', req)
-    if (!app.checkResponse(res)) return
-    // If the wallets are not open locally, they must have been opened during
-    // ordering. Grab updated info.
-    const baseWallet = app.walletMap[market.base.id]
-    const quoteWallet = app.walletMap[market.quote.id]
-    if (!baseWallet.open || !quoteWallet.open) {
-      this.balanceWgt.updateAsset(market.base.id)
-      this.balanceWgt.updateAsset(market.quote.id)
```

Comment:
This isn't relevant after #1007 which handles this for both wallets using notification.